app/components/provider_interface/application_choice_header_component.html.erb

```diff
@@ -17,9 +17,9 @@
           </h2>
           <p class="govuk-body">
             <% if time_is_today_or_tomorrow?(application_choice.reject_by_default_at) -%>
-              You have until <%= time_today_or_tomorrow(application_choice.reject_by_default_at) %> to respond to this application. Otherwise it will be automatically rejected.
```

Comment:
Is `time_today_or _tomorrow` still used anywhere? It can potentially be removed from view helper now we've moved towards showing the date and time.

Reply:
I think this may have been addressed in the latest changes pushed

Reply:
Ah, I expected for us to be able to delete the method after this change.

Have we missed a matching change here? https://github.com/DFE-Digital/apply-for-teacher-training/pull/4484/files#diff-84a45f10edd6890b4815db97d0541ba6d809a6009079d3fcbaef1e3cbeb281e8R51